### PR TITLE
Allows Tactical Shotguns to be equiped on the back slot

### PR DESCRIPTION
**Pull request by Metroidlover**
## About The Pull Request

Allows the Tactical Shotgun and Double Barrel Shotgun to be equipped on the back slot, like the regular Pump Shotgun. And gives the Tactical Shotgun a unique back sprite
## Why It's Good For The Game

IT doesn't make too much sense for other long shotguns to not be able to go on the back slot when the Pump can.
## Changelog
:cl:MetroidLover
tweak: Tactical Shotguns and Double Barrel Shotguns can now go on your back
/:cl:

### CODE REVIEW

**Comment by Ghommie** [comment]
While you are at it, can you also apply this to the double barrelled shotgun too? Their sawn off subtypes overrides it, so don't worry.

**Comment by Rohesie** [comment]
Do they have back icons?

**Comment by Metroidlover** [comment]
All shotguns already have back icons, but they just look like the pump shotgun

File: code/modules/projectiles/updated_projectiles/guns/shotguns.dm [hunk 1]
```diff
@@ -210,6 +210,7 @@ can cause issues with ammo types getting mixed up during the burst.
 /obj/item/weapon/gun/shotgun/combat
 	name = "\improper MK221 tactical shotgun"
 	desc = "The Nanotrasen MK221 Shotgun, a semi-automatic shotgun with a quick fire rate."
+	flags_equip_slot = SLOT_BACK
```
Comment (Ghommie) [inline code/modules/projectiles/updated_projectiles/guns/shotguns.dm]:
```suggestion
	flags_equip_slot = ITEM_SLOT_BACK
```

File: code/modules/projectiles/updated_projectiles/guns/shotguns.dm [hunk 2]
```diff
@@ -247,7 +248,8 @@ can cause issues with ammo types getting mixed up during the burst.
 
 /obj/item/weapon/gun/shotgun/double
 	name = "double barrel shotgun"
-	desc = "A double barreled shotgun of archaic, but sturdy design. Uses 12 Gauge slugs, but can only hold 2 at a time."
+	desc = "A double barreled shotgun of archaic, but sturdy design. Uses 12 Gauge Special slugs, but can only hold 2 at a time."
+	flags_equip_slot = SLOT_BACK
```
Comment (Ghommie) [inline code/modules/projectiles/updated_projectiles/guns/shotguns.dm]:
```suggestion
	flags_equip_slot = ITEM_SLOT_BACK
```
I too fell on this mistake in the past.

**Comment by Rohesie** [comment]
Code-wise this is fine. The question is on balance and on whether we want these shotguns to go in the back while using the sprites of another gun.

**Comment by Metroidlover** [comment]
Unfortunately, I'm not so good at spriting. Anyone who would be able to help?

**Comment by Metroidlover** [comment]
If the new sprites aren't good I can spend more time on them or get someone who is good at this to help me.

**Pull request merged**